osudroidresource/src/main/res/values-pt-rBR/options.xml

```diff
@@ -4,61 +4,225 @@
         <item>Alta</item>
         <item>Média</item>
         <item>Baixa</item>
-        <item>Desativado</item>
+        <item>Desabilitado</item>
+    </string-array>
+    <string-array name="error_meter_display_names">
+        <item>Sempre</item>
+        <item>Apenas nos replays</item>
+        <item>Desabilitado</item>
+    </string-array>
+    <string-array name="spinner_style_names">
+        <item>Moderno</item>
+        <item>Clássico</item>
+    </string-array>
+    <string-array name="metronome_switch_names">
+        <item>Desligado</item>
+        <item>Apenas no mod nightcore</item>
```

Comment:
I think this was supposed to be ``modo``, not ``mod``.

Reply:
Nah, it is mods: from "modification"